.gitignore

```diff
@@ -59,3 +59,6 @@ temp/
 ._*
 .Trashes
 .svn
+
+# Possible symlinks to wp-env install-path directories.
+/wp-*
```

Comment:
In order to have WP core files indexed in my project, I create symlinks to the directories for the wp-env. For example:

```bash
install_path=$(npm --silent run wp-env install-path)
ln -s "$install_path/WordPress" wp-core
ln -s "$install_path/WordPress-PHPUnit/tests/phpunit" wp-tests-phpunit 
```

Reply:
If you are using IntelliJ/PhpStorm, have you tried adding those as Include Paths instead (Preferences | Languages & Frameworks | PHP)?

Otherwise maybe this change is better suited for your own global `.gitignore` file.

Interesting though that you point to files from wp-env for this, I usually just point to a local checkout of the `WordPress-develop` repo.

Reply:
Yeah, I've done that in the past but this has seemed a bit simpler, and it doesn't depend on having to add the include paths to the settings. Nevertheless, I'm happy to switch to using include paths this if it's only useful to me. I've reverted in 73dbfd1675d114f8989ef6e41ce1a38f914278f6. 

> Interesting though that you point to files from wp-env for this, I usually just point to a local checkout of the `WordPress-develop` repo.

But then the files aren't the actual files in the environment, right? Then you can't hack on the core files for debugging, right?

Reply:
Actually, it's much nicer to have those symlinks after trying without. I can easily navigate to `wp-content/plugins`, for example, to drop in something for testing, for example.

I'm now just putting those symlinks inside of `vendor`.